packages/backend/src/services/UnfurlService/UnfurlService.ts

```diff
@@ -186,16 +187,6 @@ export class UnfurlService {
     }
 
     private async parseUrl(linkUrl: string): Promise<ParsedUrl> {
-        if (!linkUrl.startsWith(this.lightdashConfig.siteUrl)) {
```

Comment:
Deleted in purpose. URL validation happens on the SLack APP level. and each domain only has 1 app